libraries/ble/test/iot_test_ble_end_to_end.c

```diff
@@ -59,21 +58,6 @@ static bool bBLEConnected = false;
 static IotSemaphore_t bleConnectionSem;
 
 
-/**
- * @brief Declares and runs an MQTT system test.
- */
-#define RUN_MQTT_SYSTEM_TEST( name )                               \
```

Comment:
Why do we remove BLE test group? the library is relocated to folder /libraries/ble, not removed completely from the mainline.

Reply:
Those tests don't exist in the `libraries/ble` directory, they reference existing mqtt and shadow system tests that are in `libraries/c_sdk`

Reply:
BLE system tests (iot_test_ble_end_to_end.c) uses old MQTT, and iot_mqtt_ble_system_test.c uses coreMQT, are they equivalent and should we simply remove ipt_Test_ble_end_to_end.c?

Reply:
https://github.com/aws/amazon-freertos/blob/main/libraries/ble/test/iot_test_ble_end_to_end.c

Reply:
I would still keep that test file but remove old MQTT and Shadow test groups. There is a test which tests basic connectivity with BLE which is still useful. So the change looks good to me.

Reply:
OK. Thank you!